ansible/playbooks/utils/check_volume.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: Checking for necessary --extra_vars
+  hosts: all
+  vars:
+    example:
+      - "-e \"{'VOLUME_DEVICE':'/vol1', 'VOLUME_MKFS_DEVICE':True, 'VOLUME_DEVICE_TYPE': 'ext4'}\""
+
+  tasks:
+
+    - fail: msg="Must set VOLUME_DEVICE. Pass a value in via `-e` as JSON {{example[0]}}"
```

Comment:
👍 good suggestions here.

w.r.t. the question about `example`, presumably you could have more than one example, but I carried this along from other places in the code and figured it was some form of standard.

If there is no such standard, perhaps we should write up a ticket to remove all such instances from the codebase?